app/javascript/react/containers/MuralShowContainer.js

```diff
@@ -36,15 +80,96 @@ class MuralShowContainer extends Component {
       .then(response => response.json())
       .then(body => {
         this.setState({ mural: body });
+        this.getReviews()
+      })
+      .catch(error => console.error(`Error in fetch: ${error.message}`));
+
+    fetch(`/api/v1/user`)
+      .then(response => {
+          if (response.ok) {
+          return response;
+        } else {
+          let errorMessage = `${response.status} (${response.statusText})`,
+              error = new Error(errorMessage);
+          throw(error);
+        }
+      })
+      .then(response => response.json())
+      .then(body => {
+        this.setState({ currentUser: body });
       })
       .catch(error => console.error(`Error in fetch: ${error.message}`));
+    }
```

Comment:
I know I had mentioned that this could work, but it would be more ideal to add another .then after line 82, and put your getReviews there, then have ANOTHER .then, and place the user fetch there as well, that way you dont have multiple fetches running at once (which will happen with this implementation